Framework/Algorithms/src/FindPeaks.cpp

```diff
@@ -1087,7 +1087,7 @@ int FindPeaks::findPeakBackground(const MatrixWorkspace_sptr &input,
         "No 7th column for use FindPeakBackground result or not. ");
 
   if (peaklisttablews->rowCount() > 0) {
-    int fitresult = peaklisttablews->Int(0, 6);
+    fitresult = peaklisttablews->Int(0, 6);
```

Comment:
@FedeMPouzols @peterfpeterson Thanks. I reverted the change and added a comment referencing the issue number above.